drivers/sensor/lis2dw12/Kconfig

```diff
@@ -59,4 +59,30 @@ config LIS2DW12_TAP
 
 endif # LIS2DW12_TRIGGER
 
+config LIS2DW12_LOW_NOISE
+	bool "Low-noise configuration"
+
+config LIS2DW12_BW_FILT
+	bool "Digital filtering bandwidth selection"
+	help
+	  Specify BW_FILT value to be used in the DeviceTree.
```

Comment:
What's going on here? This is in Kconfig, not DT.

Reply:
Yes, everything should be done in DT. I don't see changes in the dts/binding. Am I missing something?